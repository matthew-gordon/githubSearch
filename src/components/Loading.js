import React from 'react'
import { ActivityIndicator, StyleSheet, View } from 'react-native'

export default function Loading({ size, ...props }) {

  return (
    <View style={[styles.loadingContainer, props.style]}>
      <ActivityIndicator size={size} />
    </View>
  )
}

const styles = StyleSheet.create({
  loadingContainer: {
    flex: 1,
    justifyContent: 'center',
    alignItems: 'center',
  },
})
