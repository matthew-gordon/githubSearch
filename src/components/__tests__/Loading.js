import React from 'react'
import { render } from '@testing-library/react-native'

import Loading from '../Loading'

describe('Input', () => {
  test('renders without crashing', async () => {
    const { baseElement } = render(<Loading />)

    expect(baseElement).toMatchSnapshot()
  })
})
