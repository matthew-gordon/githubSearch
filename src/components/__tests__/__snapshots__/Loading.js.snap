// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input renders without crashing 1`] = `
<View
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    collapsable={true}
    pointerEvents="box-none"
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "justifyContent": "center",
        }
      }
    >
      <ActivityIndicator
        animating={true}
        color="#999999"
        hidesWhenStopped={true}
        size="small"
      />
    </View>
  </View>
</View>
`;
